File: app/src/main/java/com/example/myanimedata/api/AnimeRoleResult.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class AnimeRoleResult {
    @SerializedName("mal_id")
    private final int id;

    @SerializedName("url")
    private final String url;

    @SerializedName("title")
    private final String title;

    public int getId() {
        return id;
    }

    public String getUrl() {
        return url;
    }

    public String getTitle() {
        return title;
    }

    public AnimeRoleResult(int id, String url, String title) {
        this.id = id;
        this.url = url;
        this.title = title;
    }
}

File: app/src/main/java/com/example/myanimedata/api/AnimeRoleResult.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class AnimeRoleResult(
    @field:SerializedName("mal_id") val id: Int? = 0,
    @field:SerializedName("title") val title: String? = null
) : Parcelable